projects/components/src/modal/modal-container.component.scss

```diff
@@ -26,6 +26,11 @@
     width: 640px;
   }
 
+  &.modal-size-large-tall {
```

Comment:
We probably need to include Response size. We have done it for sheet and it would be useful to add it here. It is straightforward.